lib/act-fluent-logger-rails/logger.rb

```diff
@@ -74,7 +74,12 @@ def add(severity, message = nil, progname = nil, &block)
 
     def add_message(severity, message)
       @severity = severity if @severity < severity
-      @messages << message
+      @messages << encode_message(message)
```

Comment:
Thank you for your pull request!

```
Rails.logger.info('あ'.freeze)
=> RuntimeError: can't modify frozen String
```

And I would not like to support a ruby which has no force_encoding.
What do you think the following code?

```
if message.encoding == Encoding:UTF-8
  @messages << message
else
  @messages << message.dup.force_encoding(Encoding::UTF_8)
end
```